pkg/maintainer/spv/moving_funds_test.go

```diff
@@ -38,7 +38,7 @@ func TestSubmitMovingFundsProof(t *testing.T) {
 	spvChain := newLocalChain()
 
 	// Take an arbitrary moving funds transaction:
-	// https://live.blockcypher.com/btc-testnet/tx/e6218018ed1874e73b78e16a8cf4f5016cbc666a3f9179557a84083e3e66ff7c
+	// https://live.blockcypher.com/btc-testnet/tx/e6218018ed1874e73b78e16a8cf4f5016cbc666a3f9179557a84083e3e66ff7c/
```

Comment:
Looks like an accidental change.

Reply:
This change was intentional.
I've noticed that the URLs in the file ended with `/` so I added it here.